opengrok-indexer/src/main/java/org/opengrok/indexer/util/Executor.java

```diff
@@ -210,7 +210,7 @@ public void run() {
                 timer = new Timer();
                 timer.schedule(new TimerTask() {
                     @Override public void run() {
-                        LOGGER.log(Level.WARNING,
+                        LOGGER.log(Level.FINEST,
                             String.format("Terminating process of command %s in directory %s " +
```

Comment:
Is "terminating" a keyword?

Reply:
"terminate" is a keyword, but "terminating" is not a keyword.